content/en/docs/faq.md

```diff
@@ -75,3 +75,34 @@ Not natively supported but a user can configure `revocationValidations` to `skip
+
+**Q: How are Notary and Notary Project related?**
+
+**A:** Notary is one of the subprojects under the Notary Project organization. Notary uses The Update Framework (TUF) to implement client and server components that run and interact with trusted collections that describe the content in a container registry. The name Notary comes from the `notary` CLI that is used to manage the trusted collections. The code that has the implementation of the client and the server components is available in the [notaryproject/notary](https://github.com/notaryproject/notary) repository.
+
+**Q: What is the difference between Notary Project and TUF?**
+
+**A:** [The Update Framework (aka TUF)](https://github.com/theupdateframework) is a CNCF graduated project that helps developers maintain the security of software update systems. TUF is a separate community and separate GitHub organization. One of the subprojects under Notary Project, [`notary`](https://github.com/notaryproject/notary), uses TUF for the implementation. 
+
+**Q: What is Notary Project specification?**
+
+**A:** Due to some portability challenges with the TUF-based implementation in Notary, circa 2019, the Notary Project community decided to work on a portable signature specification. This resulted in the creation of the [specifications](https://github.com/notaryproject/specifications) subproject. The Notary Project specifications are shared across repositories under Notary Project as well as used by other open-source projects and/or vendor tools that want to interoperate with the Notary Project tooling. The Notary Project community plans to add other specifications in the future as our work on software supply chain evolves.
+
+**Q: Does Notary Project signature specification leverage TUF?**
+
+**A:** No, the Notary Project [signature specification](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/signature-specification.md) does not leverage TUF.
+
+**Q: What is the difference between Notation and Notary?**
+
+**A:** [Notation](https://github.com/notaryproject/notation) and [Notary](https://github.com/notaryproject/notary) are both subprojects of Notary Project and offer capabilities for signing container images. Notary is based on [The Update Framework](https://theupdateframework.com) (TUF) and does not implement any of the Notary Project specifications while Notation implements the new [Notary Project signing specification and workflows](https://github.com/notaryproject/specifications/). While Notary has a server and client components, Notation has a CLI and libraries and leverages existing key management infrastructure and [OCI-compliant](https://opencontainers.org/) registries. An example of a `notary` implementation is Docker Content Trust (DCT). In addition to signing artifacts, Notation handles artifact verification, signature portability, and integration with third-party key/certificate management solutions via a plugin model.
```

Comment:
```suggestion
**A:** [Notation](https://github.com/notaryproject/notation) and [Notary](https://github.com/notaryproject/notary) are both subprojects of Notary Project and offer capabilities for signing container images. Notary has a server and client components, Notation has a CLI and libraries, and leverages existing key management infrastructure and [OCI-compliant](https://opencontainers.org/) registries. An example of a `notary` implementation is Docker Content Trust (DCT). Notary is based on [The Update Framework](https://theupdateframework.com) (TUF) and does not implement any of the Notary Project specifications, while Notation implements the new [Notary Project signing specification and workflows](https://github.com/notaryproject/specifications/). In addition to signing artifacts, Notation handles artifact verification, signature portability, and integration with third-party key/certificate management solutions via a plugin model.
```

Reply:
This suggestion comes from @gokarnm as mentioned in https://github.com/notaryproject/notaryproject.dev/pull/328#discussion_r1289487767

Reply:
@FeynmanZhou You suggestion is till Notary comes the first, please take a look at the latest update. Thanks.